sdk/test-utils/test-utils-vitest/src/azureAssert.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "vitest";
+import { OperationTracingOptions } from "@azure/core-tracing";
+import { supportsTracing } from "./tracing/azureTraceAssert.js";
+
+/**
+ * Augments Chai with support for Azure specific assertions.
+ *
+ * Sample usage:
+ *
+ * ```ts
+ * import chai from "chai";
+ * import { chaiAzure } from "@azure-tools/test-utils";
+ * chai.use(chaiAzure);
+ *
+ * it("supportsTracing", async () => {
+ *   await assert.supportsTracing((updatedOptions) => myClient.doSomething(updatedOptions), ["myClient.doSomething"]);
+ * });
+ * ```
+ * @param chai - The Chai instance
+ */
+export function chaiAzure(chai: Chai.ChaiStatic): void {
```

Comment:
Is this accurate? I assumed we're not using chai in this package or exporting chai assertions, but then I realized that vitest has a compatible API for extensions so maybe this is correct? Anyway would love to learn more - maybe this needs to be ported over to vitest language?

Reply:
Yes, both ways are accurate, either through the `azureMatchers.ts` which is the preferred way via Jest, or the Chai way through the `ChaiStatic`.  See the [Extending Matchers](https://vitest.dev/guide/extending-matchers.html) for guidance as it allows for both.